File: work-manage-pro-api/work-manage-pro-api/src/main/java/com/workmanagepro/workmanageproapi/domain/model/UserModel.java
package com.workmanagepro.workmanageproapi.domain.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class UserModel {

    private Long id;
    private String name;
    private String email;
    private String password;
    private LocalDateTime creationDate;

}

File: work-manage-pro-app/src/app/models/user.model.ts
export class User {
  id: number;
  name: string;
  email: string;
  password: string;
  creationDate: string;
  authStatus : string;
  constructor(id?: number, name?: string, email?: string, password?: string, creationDate?: string, authStatus?: string) {
    this.id = id || 0;
    this.name = name || '';
    this.email = email || '';
    this.password = password || '';
    this.creationDate = creationDate || '';
    this.authStatus = authStatus || '';
  }
}
